Added Help and Defaults

jfgordon2 opened the pull request:
- Add `uni --help` to provide available use options in CLI
- Assume default recipe owner if not provided (since there's currently only one, anyway) for ease of typing -- allows typing just `uni bake yarn-install` for example

![image](https://user-images.githubusercontent.com/55799997/79211103-6015c180-7e0b-11ea-9cc4-cc0cd16203f9.png)

jfgordon2 committed:
Added Help and Defaults

Added help to see options in CLI, and set default recipe owner (since there's currently only one, anyway)

daltonmenezes commented on uni:
```diff
@@ -59,6 +59,34 @@ while [ "$#" -gt 0 ]; do
+${STYLE_BOLD}  rm, remove ${STYLE_YELLOW}<package>${RESET_STYLE}        Removes one or more packages
+${STYLE_BOLD}  rm-me, remove-me${RESET_STYLE}            Removes UNI from the system
+
+${STYLE_LIGHT_CYAN}Available Recipes (by Dalton Menezes):${RESET_STYLE}
+${STYLE_YELLOW}  atom-install${RESET_STYLE}                Installs the latest version of Atom Editor
+${STYLE_YELLOW}  telegram-install${RESET_STYLE}            Installs the latest version of Telegram
+${STYLE_YELLOW}  node-install${RESET_STYLE}                Installs the nodejs latest LTS available version and npm
+${STYLE_YELLOW}  yarn-install${RESET_STYLE}                Installs the latest version of Yarn
+${STYLE_YELLOW}  stylishbuntu-theme${RESET_STYLE}          Theme Package
+${STYLE_YELLOW}  nebulosa-theme${RESET_STYLE}              Theme Package
+${STYLE_YELLOW}  mac-theme${RESET_STYLE}                   Theme Package
```
This shouldn't be hard-coded. Can you fix this bringing this information from GitHub?
In this case, it should fetch from [here](https://github.com/uni-linux/recipes/tree/master/src/daltonmenezes)

jfgordon2 replied:
I've updated to grab the README from that github with minor transformation for readability

daltonmenezes reviewed (changes requested):
Thanks @Jamesits , great additions. I made a change request, also can you update the docs?

jfgordon2 committed:
Separated recipes

Removed hardcoded recipes, moved into wget to grab current README on recipe's github

daltonmenezes reviewed (commented):
@Jamesits  It will print the following result
![img](https://i.imgur.com/5kx5QeT.png)

What if you achieve the following result? It looks cleaner and easier to read / understand. What do you think about it?
![img](https://i.imgur.com/kg98VSW.png)

jfgordon2 committed:
Update uni

Update to further trim README

jfgordon2 commented:
Updated -- just as an FYI, the more text transformation we apply on the README, the more likely it may break if styling on the markdown is significantly updated, but I've tried to keep it as flexible as possible.

![image](https://user-images.githubusercontent.com/55799997/79287833-a9f1bc80-7e8a-11ea-8b11-21c5799fd0b9.png)

jfgordon2 committed:
Update uni

Spellcheck

daltonmenezes commented:
@jfgordon2 Looks great for me, thanks. I'll keep the same style in that readme. Just a few fixes more. `uni bake how yarn-install`is not affected by the default recipe feature. Can you please fix it? After this, just update [this](https://github.com/uni-linux/uni/blob/master/README.md) and [this](https://github.com/uni-linux/uni/blob/master/docs/en/API.md) with your modifications and I can merge your pull request.

jfgordon2 committed:
Doc Updates

Updated Docs and also bugfix to include default in bake how

jfgordon2 commented:
@daltonmenezes updated for your review

daltonmenezes commented:
@jfgordon2 Well done. Thank you so much. :D
